src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -80,10 +95,20 @@ static void OnSearchHandlerPropertyChanged(BindableObject bindable, object oldVa
 				SetInheritedBindingContext(newHandler, bindable.BindingContext);
 		}
 
-		/// <summary>Bindable property for attached property <c>FlyoutItemIsVisible</c>.</summary>
+		/// <summary>
+		/// The flyout item visibility.
+		/// Flyout items are visible in the flyout by default.
+		/// </summary>
 		public static readonly BindableProperty FlyoutItemIsVisibleProperty =
 			BindableProperty.CreateAttached("FlyoutItemIsVisible", typeof(bool), typeof(Shell), true, propertyChanged: OnFlyoutItemIsVisibleChanged);
 		public static bool GetFlyoutItemIsVisible(BindableObject obj) => (bool)obj.GetValue(FlyoutItemIsVisibleProperty);
+
+		/// <summary>
+		/// Sets the visibility of Flyout items.
+		/// Flyout items are visible in the flyout by default. However, an item can be hidden in the flyout with the FlyoutItemIsVisible property.
+		/// </summary>
+		/// <param name="obj">The BindableObject that sets the visibility of Flyout items.</param>
+		/// <param name="isVisible">Set the visibility of Flyout items.</param>
```

Comment:
Boolean parameters follow a formulaic form of being written. Follow the information here: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors